src/test/java/it/gov/pagopa/pu/registry/mapper/debtposition/DebtPositionRegistryMapperServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.pu.registry.mapper.debtposition;
+
+import it.gov.pagopa.pu.registry.event.payments.dto.*;
+import it.gov.pagopa.pu.workflowhub.dto.generated.DebtPositionDTO;
+import it.gov.pagopa.pu.workflowhub.dto.generated.PaymentEventType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class DebtPositionRegistryMapperServiceTest {
+
+  private DebtPositionRegistryMapperService debtPositionRegistryMapperService;
+  private DebtPositionEventDTO2DebtPositionRegistryMapper debtPositionEventDTO2DebtPositionRegistryMapper;
+  private DebtPositionIoEventDTO2DebtPositionRegistryMapper debtPositionIoEventDTO2DebtPositionRegistryMapper;
+  private DebtPositionSendEventDTO2DebtPositionRegistryMapper debtPositionSendEventDTO2DebtPositionRegistryMapper;
+
+  @BeforeEach
+  void setUp() {
+    this.debtPositionEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionEventDTO2DebtPositionRegistryMapper());
+    this.debtPositionIoEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionIoEventDTO2DebtPositionRegistryMapper());
+    this.debtPositionSendEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionSendEventDTO2DebtPositionRegistryMapper());
+
+    debtPositionRegistryMapperService = new DebtPositionRegistryMapperService(
+        debtPositionEventDTO2DebtPositionRegistryMapper,
+        debtPositionIoEventDTO2DebtPositionRegistryMapper,
+        debtPositionSendEventDTO2DebtPositionRegistryMapper
+    );
+  }
+
+  @Test
+  void map_shouldReturnDebtPositionRegistry_whenEventIsDebtPositionEvent() {
+    DebtPositionEventDTO dto = new DebtPositionEventDTO();
+    dto.setEventType(PaymentEventType.DP_CREATED);
+    dto.setPayload(new DebtPositionDTO());
+    this.debtPositionRegistryMapperService.map(dto);
+    Mockito.verify(debtPositionEventDTO2DebtPositionRegistryMapper, Mockito.times(1)).map(Mockito.any(DebtPositionEventDTO.class));
+  }
```

Comment:
you are not checking the result!

using a mock instead of a spy, this could simpler
```suggestion
    // Given
    DebtPositionEventDTO dto = new DebtPositionEventDTO();
    dto.setEventType(PaymentEventType.DP_CREATED);
    dto.setPayload(new DebtPositionDTO());
    
    DebtPositionRegistry expectedResult = new DebtPositionRegistry();
    
    Mockito.when(debtPositionEventDTO2DebtPositionRegistryMapperMock.map(Mockito.same(dto))
       .thenReturn(expectedResult);
    
    // When
    DebtPositionRegistry result = this.debtPositionRegistryMapperService.map(dto);
    
    // Then
    Assertions.assertSame(expectedResult, result);
  }
```

The verify would not more be needed, because the when clause is already verifying it.